L/libcleri/build_tarballs.jl

```diff
@@ -0,0 +1,56 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libcleri"
+version = v"0.12.1"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/transceptor-technology/libcleri.git", "a8a1adc9dcf4e889a4d44c17acb20d74d0c6cfe5")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+export CFLAGS=$(pcre2-config --cflags)
+export LDFLAGS=$(pcre2-config --libs8)
+cd $WORKSPACE/srcdir
+cd libcleri/
+make -C Release -f makefile
+make -C Release -f makefile install INSTALL_PATH=${prefix}
+if [[ ${target} == *-mingw-* ]]; then
+   for f in ${prefix}/lib/libcleri.so*; do
+       ln -s $f $(echo $f | sed -e 's/.so/.dll/')
+   done
+fi
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Platform("i686", "linux"; libc = "glibc"),
+    Platform("x86_64", "linux"; libc = "glibc"),
+    Platform("aarch64", "linux"; libc = "glibc"),
+    Platform("powerpc64le", "linux"; libc = "glibc"),
+    Platform("i686", "linux"; libc = "musl"),
+    Platform("x86_64", "linux"; libc = "musl"),
+    Platform("aarch64", "linux"; libc = "musl"),
+    Platform("x86_64", "macos"; ),
+    Platform("x86_64", "freebsd"; ),
+    Platform("i686", "windows"; ),
+    Platform("x86_64", "windows"; )
+]
```

Comment:
We can build here for all platforms and see what's failing
```suggestion
platforms = supported_platforms(; experimental=true)
```